spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /users/:id/posts' do
+    let(:user) { User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.') }
+    it 'returns success http status code' do
+      get "/users/#{user.id}/posts"
+      expect(response.status).to eq(200)
+    end
+
+    it 'returns correct template' do
+      get "/users/#{user.id}/posts"
+      expect(response).to render_template('posts/index')
+    end
+
+    it 'returns body that include placeholder text' do
+      get "/users/#{user.id}/posts"
+      expect(response.body).to include('Number of posts: x')
+    end
+  end
+end
```

Comment:
Nice job so far👍🏼. currently, you have only written tests for the `index` action and you haven't written for the `show` action😉. kindly do well to write some tests for the show action as this is a requirement in this milestone👍🏼.

<img width="816" alt="image" src="https://github.com/toyman640/blog/assets/64643995/105508e7-d68f-4707-a952-8e77d38895df">